Makefile

```diff
@@ -14,11 +14,11 @@
 
 # protoc compiles the protobufs.
 protoc:
```

Comment:
Should we drop the Makefile target then?

Reply:
Can do! (But now the other makefile target is so lonely :)